client/pedersen.go

```diff
@@ -7,23 +7,48 @@ import (
 	"math/big"
 )
 
-func (c *Client) Pedersen(dlog *dlog.ZpDLog, val *big.Int) error {
-	c.handler.pedersenCommitter = commitments.NewPedersenCommitter(dlog)
+type PedersenCommonClient struct {
```

Comment:
Is there really any benefit if we use `PedersenCommonClient`? I think it only has one method, `decommit()`, but this method is not reused by anyone, since `PedersenECClient` has it redefined [here](https://github.com/mancabizjak/emmy/blob/15211dc84867321699ab4e3c6c8bf65fdbac0362/client/pedersen_ec.go#L92). Note that I might be missing something.